frontend/src/pages/servicesComponents/servicesComponents.constants.js

```diff
@@ -1,15 +1,33 @@
-export const initialServicesComponent = {
-    id: "9ab46509-8a3c-498a-998d-b40e78df5cd3",
-    servicesComponent: "SC1",
-    optional: false,
-    popStartMonth: "3",
-    popStartDay: "15",
-    popStartYear: "2024",
-    popEndMonth: "1",
-    popEndDay: "15",
-    popEndYear: "2025",
-    description:
-        "Develop a theory of change and identify ways to improve the program through continuous user feedback and engagement"
+export const SERVICE_REQ_TYPES = {
+    NON_SEVERABLE: "Non-Severable",
+    SEVERABLE: "Severable"
+};
+
+export const NON_SEVERABLE_OPTIONS = ["SC1", "SC2", "SC3", "SC4", "SC5", "SC6"];
```

Comment:
The idea was to have these be data driven; the API call should return the name needed (without needing translated).

Reply:
I think the challenge with the UI currently is we need to display something before it's created.

Reply:
like Steve said, these would be used to build the dropdowns on the FE based on the index. I imagine once we get the api calls in place further refinement will be needed.